src/instanceMethods/close.js

```diff
@@ -99,6 +87,32 @@ export function close (resolveValue) {
   delete this.params
 }
 
+const handlePopupAnimation = ({ popup, innerParams : { onAfterClose } }) => {
```

Comment:
> `{ popup, innerParams : { onAfterClose } }`

I have hard time readng this :) Can we simplify it to `handlePopupAnimation = (popup, onAfterClose) => {`?

Reply:
yes. Reading again...it's not clear that param is the After Close.